### ISSUE TITLE
Note searching doesn't consider all tags

### ISSUE THREAD

**Issue by ayrat555** [org-roam/org-roam#1728]
### Description
I just migrated to the org-roam v2. And I found that note searching can't find some notes by tags. For example:

#### Steps to Reproduce

Create two notes.

The first note
```
:PROPERTIES:
:ID:       E4F9AAA7-A49E-44DB-AD7D-0D08F4B28BF7
:END:
#+title: The Haunting of Bly Manor
#+filetags: :tv:tv_series:horror:drama:
```

The second note
```
:PROPERTIES:
:ID:       68E0EFA7-FB3A-420B-999F-DBB83E70E58B
:END:
#+title: Better call Soul
#+filetags: :tv:tv_series:dark_comedy:drama:
```

1. Search `drama`
2. Search `horror`

#### Expected Results

For drama search, both notes should be found.
For horror search, one note should be found

#### Actual Results

For drama search, only the second note is found
For horror search, nothing is found

### Environment

```
Copy info below this line into issue:
- Emacs: GNU Emacs 26.3 (build 1, x86_64-apple-darwin18.2.0, NS appkit-1671.20 Version 10.14.3 (Build 18D109))
 of 2019-09-02
- Framework: N/A
- Org: Org mode version 9.4.6 (9.4.6-gab9f2a @ /Users/ayrat/.emacs.d/elpa/org-9.4.6/)
- Org-roam: 2.0.0
```

I'm using the latest version of org-roam from melpa - `org-roam-20210801.626`

I migrated to v2 today, before migration note searching considered all tags

**Comment by emacs18** [comment]
When you "search", which command do you have in mind?

**Comment by emacs18** [comment]
I tried `org-agenda` command then hit `m` to search for tags.  That did not find the two simple files detailed above.  However if I simply add a headline within either file, then the search was successful, i.e., the search was successful on following file simply by adding a headline.

```
:PROPERTIES:
:ID:       68E0EFA7-FB3A-420B-999F-DBB83E70E58B
:END:
#+title: Better call Soul
#+filetags: :tv:tv_series:dark_comedy:drama:

* One headline
```
I have not used `#+filetags` before, so I don't know whether files without any headlines are supposed to match tags searches.

**Comment by ayrat555** [comment]
@emacs18 I'm using `org-roam-node-find`. In version 1 filetags were org_tags. It was possible to filter results by this field  without using org/org-agenda functions

**Comment by ayrat555** [comment]
It seems like it works if I leave a single filetag in these notes. Does it mean that org-roam doesn't support multiple tags for a single note?

**Comment by emacs18** [comment]
I am not familiar with org-roam version 1 behavior with regards to tags.
Also I have no idea whether `org-roam-node-find` should make use of tags.
I was assuming that the only names that `org-roam-node-find` used  were the headlines or files with ID properties or aliases.

**Comment by ayrat555** [comment]
Unfortunately, tags were one of the most useful features for me in the org-roam version 1. Maybe there's a bug in tags parsing.
I see one recent commit https://github.com/org-roam/org-roam/commit/a84da59b41842c0982aa68dd94636fdc87307778

**Comment by jethrokuan** [comment]
This is occurring because in the default `org-roam-node-display-template`, a very short string is allocated for tags, causing the rest of the tags to be truncated and therefore not available for search. This is definitely a bug that can be easily fixed, but I want to wait for #1724 so a less rebasing needs to be done.

For now, you can use this as a workaround:

```
(setq org-roam-node-display-template "${title} ${tags}")
```
